code/workspaces/common/src/config/images.js

```diff
@@ -0,0 +1,28 @@
+import { upperCase } from 'lodash';
+import data from './image_config.json';
+
+export function imageConfig() {
+  return Object(data.images);
+}
+
+export function imageList() {
+  return Object.keys(data.images);
+}
+
+export function versionList(type) {
+  const img = upperCase(type);
+  return data.images[img].versions.map(x => x.displayName);
+}
+export function image(type, version) {
+  const img = upperCase(type);
+  const imgName = data.images[img].versions.filter(x => x.displayName === version);
+  return imgName[0];
+}
+
+export function defaultImage(type) {
+  const img = upperCase(type);
+  if (data.images[img].versions.length > 1) {
+    return data.images[img].versions.filter(x => x.default)[0];
```

Comment:
Do we need to handle there not being a specified default? E.g. just take the first version if there is no specified default?

Reply:
Not sure I follow, do you mean we don't need the "default" tag at all and just assume we take the first one if nothing is specified?

Reply:
I think we should use the default tag. I was just thinking that if there was a time where a default wasn't specified or didn't make sense for some reason (perhaps if there was only a single image) we might want to make this robust to that possibility?

Reply:
Yes that makes sense now thanks, have changed it.